src/models/userModel.js

```diff
@@ -51,10 +51,17 @@ export default BaseModel.extend({
   attributes: {
     email: {
       type: 'string',
-      unique: true
+      unique: true,
+      editType: 'email'
```

Comment:
Seems this unknown `editType` key here is erroring for waterline validation. I'm not seeing that being used in the related admin-ui branch, is it needed?